scripts/release-notes/release_notes.go

```diff
@@ -95,6 +95,16 @@ func run() error {
 		return errors.Wrap(err, "open generated release notes")
 	}
 
+	// Check if we're on a tag and adapt variables if necessary
+	bundleVersion := head[:9]
+	shortHead := head[:7]
+	endRev := head
+	if !strings.Contains(tag, "dev") {
```

Comment:
we aren't using the `-dev` idiom anymore in release branches, will this set end tag for every new PR?

Reply:
We do still use it. But if we're exactly on a checked out tag, then the version will not have the `-dev` suffix. This is the indicator for us to link the actual release instead of the commit.

Reply:
for 1.13-1.17 we don't use the `-dev` suffix for version bumps (ref https://github.com/cri-o/cri-o/pull/3575/commits/a3564c52271d9641099f4477645f2fe6705e57a1). I think we need a different scheme to check if we're precisely on a version bump PR or not.

Reply:
I'm not sure If I get this correctly. We use the `-dev` for every commit which is not a tag right now:
https://github.com/cri-o/cri-o/blob/e8d1d22e7a43466627471e1cbb2189286b2a0153/scripts/latest-version/latest_version.go#L129

If you want then we can change this to something different, but in the end the `latest_version.go` has to return something that indicates that we're on a development version. (Maybe the git commit?)